Sources/RxFeedback/ObservableType+RxFeedback.swift

```diff
@@ -9,6 +9,8 @@
 import RxCocoa
 import RxSwift
 
+public typealias Observable = RxSwift.Observable
```

Comment:
Does it make more sense to put this in a separate file, e.g. named Observable+Alias.swift? WDYT?